ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -591,6 +591,8 @@ keys:
                 - ipv4
                 - ipv6
                 - layer2
+            mpass:
+              type: bool
```

Comment:
Please add your great description on this one as well.

Reply:
Done dana done done!! ;)